### DISCUSSION TITLE
Discussions now open

### DISCUSSION THREAD

**Discussion by willief** [safenetforum-community/NTracking#4] [Announcements]
Opening this up to see if it can become an acceptable alternative to the old safedev Discourse forum.

**Comment by dirvine** [comment]
Ok I will try and help out where I can here too.

**Comment by ShravanDalavi** [comment]
GitHub Discussions: That is the modern alternative to the old SafeDev Discourse forum. It's offering integrated collaboration along with easy tracking, so it remains an absolutely seamless option for development-focused conversations.
